documentation/modules/rn-2.4.adoc

```diff
@@ -0,0 +1,76 @@
+// Module included in the following assemblies:
+//
+// * documentation/doc-Release_notes/master.adoc
+
+[id="rn-24_{context}"]
+= {project-full} 2.4
+
+You can migrate virtual machines (VMs) from VMware vSphere or {rhv-full} to {virt} with {the-lc} {project-first}.
+
+The release notes describe technical changes, new features and enhancements, and known issues.
+
+[id="technical-changes-24_{context}"]
+== Technical changes
+
+This release has the following technical changes:
+
+.Technical change 1
+
+Description 1
+
+.Technical change 2
+
+Description 2
+
+[id="new-features-and-enhancements-24_{context}"]
+== New features and enhancements
+
+This release has the following features and improvements:
+
+.link:https://issues.redhat.com/browse/MTV-427[MTV-427] - OCP console plugin
+
+MTV Operator installed from CLI. +
+MTV web console URL.
+
+.link:https://issues.redhat.com/browse/MTV-426[MTV-426] - OpenStack migration
+
+See new link:https://access.redhat.com/documentation/en-us/migration_toolkit_for_virtualization/2.3/html-single/installing_and_using_the_migration_toolkit_for_virtualization/index#rhv-prerequisites_mtv[prerequisites] for RHV and VMware migration, and link:https://access.redhat.com/documentation/en-us/migration_toolkit_for_virtualization/2.3/html-single/installing_and_using_the_migration_toolkit_for_virtualization/index#migrating-virtual-machines-cli_mtv[procedure] to migrate virtual machines.
+
+[id="known-issues-24_{context}"]
+== Known issues
+
+This release has the following known issues:
+
+.link:https://issues.redhat.com/browse/MTV-505[MTV-505] - Power off OpenStack VM in cold migration
+
+This issue implements the following: +
+
+` pkg/controller/plan/adapter/openstack/client.go` +
+` // Power off the source VM.` +
+` func (c *Client) PowerOff(vmRef ref.Ref) error` +
+` { return nil }`
+
+.link:https://issues.redhat.com/browse/MTV-349[MTV-349] - RHV snapshots are not deleted after a successful migration
+
+Issue description
```

Comment:
yeah, this one should remain from 2.3:
```
Snapshots are not deleted after warm migration

The Migration Controller service does not delete snapshots automatically after a successful warm migration of a RHV VM. You can [delete the snapshots manually](https://access.redhat.com/documentation/en-us/openshift_container_platform/4.9/html-single/virtualization/index#virt-deleting-vm-snapshot-web_virt-managing-vm-snapshots). [(BZ#22053183)](https://bugzilla.redhat.com/show_bug.cgi?id=2053183)
```
(the bug should change to https://issues.redhat.com/browse/MTV-349)